addons/mail/static/src/models/composer_view.js

```diff
@@ -487,28 +485,28 @@ registerModel({
             }
         },
         /**
-         * Clears the active suggested record on closing mentions or adapt it if
-         * the active current record is no longer part of the suggestions.
+         * Clears the active suggestion on closing mentions or adapt it if
+         * the active current suggestion is no longer part of the suggestions.
```

Comment:
The first line can still hold one more word 👀

"adapt" → "adapts"?

"active current suggestion" → "active current" seems a bit redundant. Apart from that, if you decide to keep the two anyway, I'd suggest you swap their order: "current active suggestion".